dali/dfuXRefLib/dfuxreflib.cpp

```diff
@@ -335,8 +335,9 @@ static unsigned short getDafsPort(const SocketEndpoint &ep,unsigned &numfails,Cr
     }
     else if (numfails>5)
         return 0;
-    if (testDaliServixPresent(ep)) 
-        return ep.port?ep.port:getDaliServixPort();
+    unsigned short nPort = ep.port;
+    if (testDaliServixPresent(ep, &nPort))
+        return nPort?nPort:getDaliServixPort();
```

Comment:
We might possibly change epout port in checkLocalDaFileSvr() for spray() and despray().
The return here goes back through testDaliServixPresent() and back through checkLocalDaFileSvr() for srcip and dstip in spray() / despray()